libtock/nrf51_serialization.h

```diff
@@ -19,12 +19,21 @@ int nrf51_serialization_subscribe (subscribe_cb cb);
 
 // Pass a buffer for the driver to write received UART bytes to.
 __attribute__ ((warn_unused_result))
-int nrf51_serialization_setup_rx_buffer (char* rx, int rx_len);
+int nrf51_serialization_setup_receive_buffer (char* rx, int rx_len);
+
+// Pass a buffer for the driver to write received UART bytes to.
+__attribute__ ((warn_unused_result))
+int nrf51_serialization_setup_write_buffer (char* rx, int rx_len);
```

Comment:
This no longer seems to be implemented in the c file

Reply:
Fixed.